examples/circle_packing.py

```diff
@@ -1,36 +1,40 @@
-import cvxpy as cp, numpy as np, cvxopt, matplotlib.pyplot as plt
-import ncvx
+import cvxpy as cp
+import numpy as np
+import matplotlib.pyplot as plt
+import ncvx as nc
 
-np.random.seed(1)
+# This example is described in Section 6.3 of the NCVX paper.
 
-N = 41 # number of circles
-r = 0.4 + 0.6 * np.random.rand(N) # radii
+rng = np.random.default_rng(seed=123)
 
-#define variables.
+N = 41                         # number of circles
+r = 0.2 + 0.3 * rng.random(N)  # radii
+
+# Define variables.
 x_vals = [cp.Variable() for i in range(N)]
 y_vals = [cp.Variable() for i in range(N)]
 
-objective = cp.Minimize(cp.maximum( *(x_vals + y_vals) ) + 0.5)
+objective = cp.Minimize(cp.maximum(*(x_vals + y_vals)) + 0.5)
 constraints = []
 for i in range(N):
-    constraints  += [0.5*r[i] <= x_vals[i],
-                     0.5*r[i] <= y_vals[i]]
+    constraints += [r[i] <= x_vals[i],
+                    r[i] <= y_vals[i]]
 diff_vars = []
-for i in range(N-1):
-    for j in range(i+1,N):
+for i in range(N - 1):
+    for j in range(i + 1, N):
         t = cp.Variable()
-        diff_vars.append(ncvx.Annulus(2, 0.5 * (r[i]+r[j]), N))
+        diff_vars.append(nc.Annulus(2, r[i] + r[j], N))
```

Comment:
What happened to the 0.5*?

Reply:
Same as one above; I think the original code was using a diameter in the ``r`` variable.